### PR TITLE
[refactor] some code cleanup for min/max function.

### PR DESCRIPTION
**Pull request by zhannngchen**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/exprs/aggregate_functions.cpp [hunk 1]
```diff
@@ -654,22 +654,6 @@ void AggregateFunctions::min(FunctionContext*, const DecimalV2Val& src, DecimalV
     }
 }
 
-template <>
-void AggregateFunctions::min(FunctionContext*, const LargeIntVal& src, LargeIntVal* dst) {
-    if (src.is_null) {
```
Comment (yiguolei) [inline be/src/exprs/aggregate_functions.cpp:659]: why delete this function? LargeIntValue is not used?

Comment (zhannngchen) [reply be/src/exprs/aggregate_functions.cpp:659]: The implementation for LargeIntValue is totally duplicate with the template function of min/max, I removed these code and add a template instance for LargeIntValue type, see L2543 and L2593

**Comment by zhannngchen** [comment]
@yiguolei @morningman 
Do you have time to help to review this PR?

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**